### PR TITLE
Contrcut MDLCSR

### PR DESCRIPTION
**Pull request by jeageun**
# Description
Cleaned version of MDLCSR construction.

## PR description

This PR _write a high-level description of your pull request_

## Checklist

- [ ] The additions follow the code standards in the [developer guide](pando-rt/docs/developer.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.

### CODE REVIEW

File: include/pando-lib-galois/graphs/graph_traits.hpp [hunk 1]
```diff
@@ -196,5 +196,26 @@ struct graph_checker {
       sizeof(addEdgesTopologyOnly(0)) == sizeof(Yes) && sizeof(addEdges(0)) == sizeof(Yes) &&
       sizeof(deleteEdges(0)) == sizeof(Yes);
 };
+
+/**
+ * @brief this is the graph interface, methods from here should mostly be used
```
Comment (AdityaAtulTewari) [inline include/pando-lib-galois/graphs/graph_traits.hpp:201]: Please fix this comment.

File: test/import/test_wmd_importer.cpp [hunk 2]
```diff
@@ -270,3 +271,107 @@ INSTANTIATE_TEST_SUITE_P(
...
+                      std::make_tuple("/pando/graphs/rmat_571919_seed1_scale10_nV1024_nE10447.el",
+                                      1024)));
+
+INSTANTIATE_TEST_SUITE_P(
+    DISABLED_BigFiles, MirrorDLCSRInitEdgeList,
+    ::testing::Values(
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale11_nV2048_nE22601.el", 2048),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale12_nV4096_nE48335.el", 4096),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale13_nV8192_nE102016.el", 8192),
+        std::make_tuple("/pando/graphs/rmat_571919_seed1_scale14_nV16384_nE213350.el", 16384)));
```
Comment (AdityaAtulTewari) [inline test/import/test_wmd_importer.cpp:377]: Have you tested with these files?

File: test/test_mirror_master_table.cpp [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  runTest(filepath, numVertices);
+  return 0;
+}
+
+void runTest(const char* elFile, std::uint64_t numVertices) {
+  using ET = galois::ELEdge;
+  using VT = galois::ELVertex;
+  using Graph = galois::MirrorDistLocalCSR<VT, ET>;
+  galois::HostLocalStorageHeap::HeapInit();
+  galois::PodLocalStorageHeap::HeapInit();
```
Comment (AdityaAtulTewari) [inline test/test_mirror_master_table.cpp:71]: These should come right after pandoMain.

**Review by AdityaAtulTewari — CHANGES_REQUESTED** [review]
We want to get the python testing pr done first.

File: test/import/test_wmd_importer.cpp (hunk 2 above)
Comment (jeageun) [reply test/import/test_wmd_importer.cpp:377]: For 2048, yes, but not for the others.

Comment (AdityaAtulTewari) [reply test/import/test_wmd_importer.cpp:377]: just deleted disabled and run it locally. If that works then we are good.

File: test/test_mirror_master_table.cpp [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+template <typename T>
+using V = pando::Vector<T>;
+
+template <typename T>
+using G = pando::GlobalPtr<T>;
+void runTest(const char* elFile, std::uint64_t numVertices);
+
+int pandoMain(int argc, char** argv) {
+  galois::HostLocalStorageHeap::HeapInit();
+  galois::PodLocalStorageHeap::HeapInit();
```
Comment (AdityaAtulTewari) [inline test/test_mirror_master_table.cpp:40]: These calls should only run on host 0.

File: test/import/test_wmd_importer.cpp (hunk 2 above)
Comment (jeageun) [reply test/import/test_wmd_importer.cpp:377]: UBSAN and Release passed.

DRVx takes too long...

Comment (AdityaAtulTewari) [reply test/import/test_wmd_importer.cpp:377]: drvx we can ignore for now.

File: include/pando-lib-galois/graphs/graph_traits.hpp [hunk 5]
```diff
@@ -196,5 +196,26 @@ struct graph_checker {
       sizeof(addEdgesTopologyOnly(0)) == sizeof(Yes) && sizeof(addEdges(0)) == sizeof(Yes) &&
       sizeof(deleteEdges(0)) == sizeof(Yes);
 };
+
+/**
+ * @brief this is the gluon_graph interface, methods from here should mostly be used
```
Comment (AdityaAtulTewari) [inline include/pando-lib-galois/graphs/graph_traits.hpp:201]: "should mostly be used" doesn't mean much.

**Review by AdityaAtulTewari — APPROVED** [review]

**Comment by AdityaAtulTewari** [comment]
Please wait for ci to merge

**Comment by jeageun** [comment]
following tests FAILED:
	252 - PodSpecificStorage.RemoteWriteLocalWriteBothRead (Failed)
It is not my code.... :( not sure...

**Pull request merged**